lib/ember-cli-rails.rb

```diff
@@ -1,3 +1,4 @@
+require 'fileutils'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.